File: Retrofit/app/src/main/java/com/cyq/retrofit/Person.java
package com.cyq.retrofit;

/**
 * @author : ChenYangQi
 * date   : 1/28/21 23:25
 * desc   :
 */
public class Person {

    private String phone;
    private String name;
    private Integer id;
    private Integer age;
    private String email;

    public String getPhone() {
        return phone;
    }

    public void setPhone(String phone) {
        this.phone = phone;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public Integer getAge() {
        return age;
    }

    public void setAge(Integer age) {
        this.age = age;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    @Override
    public String toString() {
        return "Person{" +
                "phone='" + phone + '\'' +
                ", name='" + name + '\'' +
                ", id=" + id +
                ", age=" + age +
                ", email='" + email + '\'' +
                '}';
    }
}

File: Retrofit/app/src/main/java/com/cyq/retrofit/Person.kt
package com.cyq.retrofit

/**
 * @describe xxx
 * @author chenyq113@midea.com
 * @time 2021/1/28 17:45
 */
data class Person(
    val code: Int,
    val `data`: Data,
    val description: String
)

data class Data(
    val age: Int,
    val email: String,
    val id: Int,
    val name: String,
    val phone: String
)